datatableview/columns.py

```diff
@@ -11,7 +11,10 @@
 import django
 from django.db import models
 from django.db.models import Model, Manager, Q
-from django.db.models.fields import FieldDoesNotExist
+try:
+    from django.db.models.fields import FieldDoesNotExist
+except ImportError:
+    from django.core.exceptions import FieldDoesNotExist
```

Comment:
```suggestion
from django.core.exceptions import FieldDoesNotExist
```

Reply:
True, looks like I overlooked the fact that it was backported. Commiting your suggestion 👍